### PR TITLE
git-log: improve descs, add --grep

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

File: pages/common/git-log.md [hunk 1]
```diff
@@ -2,14 +2,18 @@
 
 > Show a history of commits.
 
-- Show a history of commits:
+- Show the sequence of commits leading to the current one:
```
Comment (agnivade) [inline pages/common/git-log.md]: I am guessing you added "leading to the current one" to give a sense of the chronological ordering. I feel "leading to" is not perfectly clear on whether it starts from top or ends at the bottom. 

How about "Show sequence of commits starting from the current one" ?

Comment (waldyrious) [inline pages/common/git-log.md]: Good point. Maybe then I'd add "in reverse chronological order" to make that explicit... what do you think?

Comment (agnivade) [inline pages/common/git-log.md]: sure, sounds good.

**Comment by waldyrious** [comment]
@agnivade PTAL

**Comment by agnivade** [comment]
perfect.

**Comment by ostera** [comment]
Not trying to cause trouble here, but using "history" is far simpler to understand than "reverse chronological order" 😛

**Comment by waldyrious** [comment]
Simpler, sure. Easier to understand? Not so sure. "History" is a very vague term, while "list of commits in reverse chronological order", despite being a mouthful, is both specific and composed of common words which most people would understand (except for perhaps chronological). So for now I stand by my choice.

**Comment by waldyrious** [comment]
Ok, I used "sequence" instead of "list". Since we're already mentioning "order", I suppose we could use "list" in the description to get a less fancy word without losing precision.

**Comment by agnivade** [comment]
@waldyrious - This is a relatively low hanging fruit. Lets get this one done whenever you have some time to look at it ? :)

**Comment by waldyrious** [comment]
I was waiting for @ostera to react :)

**Comment by waldyrious** [comment]
@ostera ping :)

**Comment by ostera** [comment]
LGTM!

**Comment by waldyrious** [comment]
@agnivade feel free to merge anytime :)

**Comment by ostera** [comment]
It needs another LGTM thou

**Comment by waldyrious** [comment]
@ostera that'd be @agnivade's (he hasn't included any of the keywords that trigger the lgtm service in his comments so far)

**Comment by waldyrious** [comment]
Hahah now it caught my own "lgtm" xD well, I guess he has already expressed his approval, so I'll go ahead and merge.

**Pull request merged**